### DISCUSSION TITLE
Enabling CoD payment

### DISCUSSION THREAD

**Discussion by jainneerajnj** [ONDC-Official/protocol-network-extension#37] [Announcements]
The draft for enabling CoD Payment on the network has been published. [Link](https://docs.google.com/document/d/1X4VyBAGc02tHgyPnqAIH5JW5BvCeHgoCOU6mGPi52iU/edit)

**Comment by BLR-0118** [comment]
@aakasjayaraj @abaweja @batramanuj @mahoriR @mherle @VidyaGF @NajeebMohammed https://github.com/orgs/ONDC-Official/teams/participants

**Comment by BLR-0118** [comment]
@dashbitla-ib - for your comments

**Reply by dashbitla-ib** [reply]
@BLR-0118 Spec is alright. Added few questions to the contract.
